### ISSUE TITLE
prado-cli: error generating AR Classes

### ISSUE THREAD

**Issue by sicutdeux** [pradosoft/prado#714]
Hi same issue related to the WSAT, trying to generate the AR Classes/Relations it throws this error:

```
* Unable to find application.xml in /Users/sicutdeux/Trunk/daptrius/prado-app/protected/vendor/bin
PHP Fatal error:  Uncaught Error: Class 'TActiveRecordManager' not found in /Users/sicutdeux/Trunk/daptrius/prado-app/protected/vendor/pradosoft/prado/bin/prado-cli.php:750
Stack trace:
#0 /Users/sicutdeux/Trunk/daptrius/prado-app/protected/vendor/pradosoft/prado/bin/prado-cli.php(104): PradoCommandLineActiveRecordGenAll->performAction(Array)
#1 /Users/sicutdeux/Trunk/daptrius/prado-app/protected/vendor/pradosoft/prado/bin/prado-cli.php(48): PradoCommandLineInterpreter->run(Array)
#2 /Users/sicutdeux/Trunk/daptrius/prado-app/protected/vendor/pradosoft/prado/bin/prado-cli(4): include('/Users/sicutdeu...')
#3 {main}
  thrown in /Users/sicutdeux/Trunk/daptrius/prado-app/protected/vendor/pradosoft/prado/bin/prado-cli.php on line 750

Fatal error: Uncaught Error: Class 'TActiveRecordManager' not found in /Users/sicutdeux/Trunk/daptrius/prado-app/protected/vendor/pradosoft/prado/bin/prado-cli.php:750
Stack trace:
#0 /Users/sicutdeux/Trunk/daptrius/prado-app/protected/vendor/pradosoft/prado/bin/prado-cli.php(104): PradoCommandLineActiveRecordGenAll->performAction(Array)
#1 /Users/sicutdeux/Trunk/daptrius/prado-app/protected/vendor/pradosoft/prado/bin/prado-cli.php(48): PradoCommandLineInterpreter->run(Array)
#2 /Users/sicutdeux/Trunk/daptrius/prado-app/protected/vendor/pradosoft/prado/bin/prado-cli(4): include('/Users/sicutdeu...')
#3 {main}
  thrown in /Users/sicutdeux/Trunk/daptrius/prado-app/protected/vendor/pradosoft/prado/bin/prado-cli.php on line 750
```

My config have this:

```
<?xml version="1.0" encoding="utf-8"?>

<application id="prado-app" mode="Debug">
	<paths>
		<using namespace="Application.Common.*" />
		<using namespace="System.Data.*" />
		<using namespace="System.Security.*" />
		<using namespace="System.Web.UI.ActiveControls.*"/>
		<using namespace="System.Wsat.*"/>
		<using namespace="System.Util.*" />
		<using namespace="System.I18N.*" />
	</paths>
	<!-- modules configured and loaded for all services -->
	<modules>
		<module id="request" class="THttpRequest" UrlFormat="HiddenPath" />
		<module id="parameter" class="TParameterModule" ParameterFile="Application.Data.Settings" />

		<module class="System.Data.ActiveRecord.TActiveRecordConfig" EnableCache="true">
    		<database ConnectionString="mysql:host=localhost;dbname=daptrius-db-testing"
        		Username="root" Password="" />
  		</module>
	</modules>
	<services>
		<!-- page service -->
		<service id="page" class="TPageService" DefaultPage="Home">
			<modules>
				<module id="session" class="THttpSession" SessionName="SSID" CookieMode="Only" UseCustomStorage="false" AutoStart="true" TimeOut="28800" />
				<module id="asset" class="System.Web.TAssetManager" />
			</modules>
			<pages MasterClass="Application.Layouts.MainLayout" />
		</service>
		<service id="wsat" class="System.Wsat.TWsatService" Password="1234" />
	</services>
</application>
```

**Comment by ctrlaltca** [comment]
Hi I've not been able to reproduce the issue, but committed a possible fix in dbe80183d57377c136249bff340c2c626c4d9aed.
If that doesn't work, can you please try adding this namespace in application.xml?
```
<using namespace="System.Data.ActiveRecord.*" />
```

**Comment by ctrlaltca** [comment]
Closing. If you still need help with this issue, feel free to reopen.
